### PR TITLE
Fix a typo shared among many installation scripts

### PR DESCRIPTION
**Pull request by hoechenberger**
`retring` -> `retrying`

Should I bump the version numbers of all features I touched here?

### CODE REVIEW

File: src/git-lfs/install.sh [hunk 1]
```diff
@@ -82,7 +82,7 @@ receive_gpg_keys() {
         echo "(*) Downloading GPG key..."
         ( echo "${keys}" | xargs -n 1 gpg --recv-keys) 2>&1 && gpg_ok="true"
         if [ "${gpg_ok}" != "true" ]; then
-            echo "(*) Failed getting key, retring in 10s..."
+            echo "(*) Failed getting key, retrying in 10s..."
```
Comment (samruddhikhandale) [inline src/git-lfs/install.sh:85]: Can we also bump patch versions for all the updated Features? Thank you!

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks for fixing the typo, left one comment!

File: src/git-lfs/install.sh (hunk 1 above)
Comment (hoechenberger) [reply src/git-lfs/install.sh:85]: Certainly, I've addressed this in [500562b](https://github.com/devcontainers/features/pull/1006/commits/500562b6394645cc4f7db3d9a3f027224da07ab5)

**Review by samruddhikhandale — APPROVED** [review]
Thank you ⚡

**Pull request merged**